### ISSUE TITLE
About "auto" button

### ISSUE THREAD

**Issue by jenrisito** [haxball/haxball-issues#66]
When I push the auto button to randomize teams, it just chooses the player who is on the top, following FIFO method (First In-First Out).

I think it's a feature who should be fixed. Thank you :)

**Comment by erenseymen** [comment]
I think FIFO method is better but it's just my opinion.

**Comment by basro** [comment]
I also think FIFO is better which is why it is like this atm, but some way to randomize players is also useful.
I'll be adding a way to select random players separate from the auto button.

**Comment by awareofdistractions** [comment]
Also changing name from "Auto" to "Top" would be more intuitive imho.
![screenshot from 2017-11-22 20 28 56](https://user-images.githubusercontent.com/12998823/33146350-1c64f476-cfc4-11e7-8c27-c57f7bc2173a.png)

**Comment by Leo-MapCreator** [comment]
@basro I agree with awareofdistractions, the name of nowadays Auto should be change to Top, to make more intuitive, plus sometimes you don't know what people asked for when they ask for Auto instead of picking players, if it was Auto from Flash or Auto from HTML5 that the person asked we will never know
